sysutils/poac/Portfile

```diff
@@ -22,6 +27,16 @@ checksums           rmd160  13108a1b06d9c9cf7d4da27d49b121d255467d62 \
                     sha256  8d7b13d2a4421e1ca1b3d6208aff67583d99b86194b92bd3be0bf8b61610a8b0 \
                     size    149703
 
+# Do not allow CMake to use system Git on older systems
+if {${os.platform} eq "darwin" && ${os.major} < 18} {
+     set cmake_git       port:git
+} else {
+     set cmake_git       bin:git:git
+}
+
+depends_fetch-append \
+                    ${cmake_git}
+
```

Comment:
If a port needs `git`, it's preferable to use the version packaged with MacPorts. So I'd recommend doing that across-the-board.

Reply:
You don't want to use a `bin`-style dependency, as that will use the system git if installed. Instead, explicitly use `port:git`.